polaris/polaris/helpers.py

```diff
@@ -259,15 +260,17 @@ def invalidate_session(request: Request):
 
 def interactive_args_validation(
     request: Request,
-) -> Tuple[Optional[Transaction], Optional[Asset], Optional[Response]]:
+) -> Tuple[Optional[Transaction], Optional[Asset], Optional[str], Optional[Response]]:
```

Comment:
Can we make this a type instead of a tuple so that we don't need to change all the caling sites everytime something is added or removed, and it will also be self-documenting since we'd have a `.callback` property instead of an `Optional[str]` property